__fixtures__/starwars/schema.graphql

```diff
@@ -44,6 +44,8 @@ enum LengthUnit {
   METER
   # Primarily used in the United States
   FOOT
+  # Ancient unit used during the Middle Ages
+  CUBIT @deprecated(reason: "Test deprecated enum case")
```

Comment:
I'm OK with this given that tests passed but I want to make sure our JS team is OK with these fixtures being updated since this could theoretically affect other things besides swift

Reply:
Makes sense, just let me know (or if there's a better way to test this without making a change to a schema used by other languages, happy to update)

Reply:
(update: They seem cool with it)